src/tests/ftest/launch.py

```diff
@@ -1105,7 +1122,7 @@ def run_gdb(pattern):
             pattern (str): the fnmatch/glob pattern of core files to
                            run gdb on
         """
-        import magic
+        import magic # pylint: disable=import-outside-toplevel
```

Comment:
(pylint-import-error) Unable to import 'magic'

Reply:
I can't seem to find a `magic` RPM for Python 3.  We really just need to bite the bullet and move linting to Fedora.